src/convert.ts

```diff
@@ -2426,13 +2424,15 @@ export default function convert(config: ConvertConfig): ESTreeNode | null {
       Object.assign(result, {
         type: AST_NODE_TYPES.TSInterfaceDeclaration,
         body: interfaceBody,
-        id: convertChild(node.name),
-        heritage: hasImplementsClause
-          ? interfaceHeritageClauses[0].types.map(el =>
-              convertHeritageClause(AST_NODE_TYPES.TSInterfaceHeritage, el)
-            )
-          : []
+        id: convertChild(node.name)
       });
+
+      if (interfaceHeritageClauses.length > 0) {
+        result.extends = interfaceHeritageClauses[0].types.map(el =>
```

Comment:
i have only one concern here

```
interface foo implements bar extends foo {}
```

its going to pick up wrong heritage

Reply:
i know that whas a case before to, but prettier is going to mess up class here

Reply:
Yep good point, I'll fix it

Reply:
there is also more broken example
```
interface foo extends bar extends baz {}
```